### PR TITLE
Patch w/ instructions to find updated IDs

### PR DESCRIPTION
**Pull request by Kalaborative**
This is a working patch as of 6/29/2018. 

Also includes instructions if the extension is broken again due to mismatching IDs and you need to find them. Hope they are clear enough for anyone else!

### CODE REVIEW

File: webpack_override.js [hunk 1]
```diff
@@ -61,7 +63,8 @@
 			};
 			
 			// DATA
-			window.gameVars.Game.GameData = getModule("989ad6");
+			window.gameVars.Game.GameData = getModule("064c0a93");
```
Comment (ma44) [inline webpack_override.js:66]: Not exactly correct due to the first thing being a ID for a localization collection

Comment (Kalaborative) [reply webpack_override.js:66]: Ah, okay. It must've been for the Korean language (although the game is still in English for me). Let me correct that and see if it still works

**Comment by reuke** [comment]
@Kalaborative , thanks for your contribution. But changing id again and again is kind of cat and mouse game.
https://github.com/reuke/surviv.io-cheat/commit/f80e85098ee0210c32c481a055f544e25b3ae6bf introduce the early version of automatic detection of desired IDs.
So i have to reject this pull request.

**Comment by Kalaborative** [comment]
That's even better thanks. I was hoping we wouldn't have to keep doing that. 😂

**Pull request closed without merging**